TypeCobol.Test/Parser/CodeElements/SORT_ko.cbl

```diff
@@ -1,13 +1,7 @@
-﻿* neither USING not GIVING
-SORT x    ASCENDING      d.
-SORT x ON DESCENDING KEY d.
-SORT x    ASCENDING      d1 d2 d3.
-SORT x    DESCENDING     d1 d2 d3.
-SORT x ASCENDING d1 d2 ASCENDING d3 DESCENDING d4 ASCENDING d5 d6 DESCENDING d7 d8 d9.
-* missing USING
+﻿* missing USING: invalid but not checked by ANTLR grammar
```

Comment:
I kept them because they are invalid and I've added a comment explaining that this is checked later. So they are either ok or ko depending on how you see them...
Do you want me to move them into _Sort_ok.cbl_ ?

Reply:
Ok keep it like that